source-academy/js-slang
# Fix missing frame

### Description

This is a fix for the [issue](https://github.com/source-academy/cadet-frontend/issues/1825)

Previously:
<img width="806" alt="Screenshot 2021-06-26 at 12 37 54 AM" src="https://user-images.githubusercontent.com/24221801/123457747-d10f5680-d616-11eb-93c4-6f50c0ba55e3.png">
Now:
<img width="702" alt="Screenshot 2021-06-26 at 12 37 26 AM" src="https://user-images.githubusercontent.com/24221801/123457758-d5d40a80-d616-11eb-95f9-c64935e0193c.png">


### How to Test
Run the following code and verify that `sum_of_squares` frame is visualized.
Click [here](https://sa-env-visualizer-beta.vercel.app/) for more extensive code samples for testing.
```
function square(x) {
    debugger;
    return x * x;
}

function sum_of_squares(x, y) {
    debugger;
    return square(x) + square(y);
}

function f(a) {
    return sum_of_squares(a + 1, a * 2);
}

f(5);
```

### Checklist
